omnibus/python-scripts/postinst.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module provides functions for managing Datadog integrations and Python dependencies after installation
+
+Usage:
+- The script should be run with a single argument specifying the installation directory.
+- Example: `python postinst.py /path/to/install/dir`
+"""
+
+import os
+import sys
+import packages
+
+def main():
+    try:
+        if len(sys.argv) != 2:
+            print("Usage: postinst.py <INSTALL_DIR>")
+        install_directory = sys.argv[1]
+        if os.path.exists(install_directory):
+            postinst_python_installed_packages_file = packages.postinst_python_installed_packages_file(install_directory)
+            packages.create_python_installed_packages_file(postinst_python_installed_packages_file)
+            flag_path = f"{install_directory}/.install_python_third_party_deps"
```

Comment:
When/why is this file supposed to be created? It doesn't seem to be written to nor read from in your PR

Reply:
This file is our feature flag.
Using a file as a flag offers greater persistence and flexibility compared to environment variables (the first option we thought), as it remains accessible across sessions and subprocesses, including during apt-get installs. Files are also easier to manage, allowing manual or programmatic creation and removal, and are more traceable for debugging. This approach avoids the risk of environment variables not being propagated correctly, making it more reliable in isolated environments.
You can see we're creating this file in our tests.

Reply:
@chouquette This file is acting like a feature flag, it's supposed to be created by users willing to use try this upgrade feature.
We couldn't use an environment variable for various reasons...

Reply:
Gotcha, thanks for the explanation!